### ISSUE TITLE
User #1, User #2 instead of the username

### ISSUE THREAD

**Issue by Hubaxis7** [strukturag/nextcloud-spreedme#55]
When you create a temporary Administrator user is required to enter his name and expiration date.
But in interface nexcloud temporary users as it displays the name of "User 1", "User 2", "User 3", etc.
[https://i.imgsafe.org/789c68bbc4.png](url)

**Comment by leonklingele** [comment]
This works fine for me.. Can you please generate a 'Temporary Password' and paste it here? (make it valid for 1 minute)

**Comment by Hubaxis7** [comment]
My cloud:
https://disk.businessvideo.by/index.php/apps/spreedme#sl
Temporary user leonklingele
https://disk.businessvideo.by/index.php/apps/spreedme?tp=MTQ3Njk3NjU2MDpleHQvbGVvbmtsaW5nZWxlLzU4MDhjNDQxMDE2MDExLjM1OTY5NTgxOjI6S1Y2cVF1NEVWZU1jOTRCTnMzYlkvV1dVN0Z3c0laZ1pBK3l1YzZndzNoWT0%3D#sl

Maybe it has something to do with the Russian localization? I use the Russian language!

**Comment by leonklingele** [comment]
Thanks @Hubaxis7!
I don’t know if that’s related to your problem, but PATCH requests seem to fail with your installation. Try it yourself:

``` bash
$ curl --request PATCH https://disk.businessvideo.by/webrtc/api/v1/sessions/test/
404 page not found
```

How the response should look line:

```
{
  "code": "session_patch_failed",
  "message": "Failed to patch session",
  "success": false
}
```

Can you please post your Apache and Spreed WebRTC config?

**Comment by Hubaxis7** [comment]
Thank you for your response. I found the problem because I did not have a name at the time of users.
In the configuration file "webrtc.conf" I did not remove the default setting and added config "Minimal Spreed WebRTC configuration for Nextcloud" from the web interface Nextcloud.
I removed the standard "webrtc.conf" options and now everything works as expected.

I also noticed that the names of users in the video is not always displayed completely. It's not very nice. As a workaround I edited the CSS (removed from class .audiovideo .remoteVideo .peerLabel property max-width: 50%;). It would be good to add in the option (Always display names | Show hover | Always hide names).
